setup.sh

```diff
@@ -84,18 +84,27 @@ setGazeboVariables() {
 }
 
 setupIdf() {
+    IDF_REQUIRED_VERSION=v5.0-beta1
+
     # We allow the user to set a global path to build dependencies that are
     # fetched autonomously, e.g., in the script ~/rofi.pre.env. If there is no
     # such path, we put the build tools into the current RoFI worktree.
+    #
+    # We also fetch the parameters when the global version doesn't match the
+    # required version
+    INSTALLED_VERSION=$(cd ${ROFI_TOOLS_PATH}/esp-idf; git describe 2>/dev/null)
     if [ ! $ROFI_TOOLS_PATH ]; then
         export ROFI_TOOLS_PATH=$ROFI_ROOT/build.deps
+    elif [ ! "${INSTALLED_VERSION}" = "${IDF_REQUIRED_VERSION}" ]; then
```

Comment:
This condition is weird - is it doing an assignment? or is it negated eq (then use `!=` or `-ne`)...

Reply:
It is negated `eq`.  You are right that `!=` is mode idiomatic.